provider/custom_server.go

```diff
@@ -162,16 +162,19 @@ func (c *CustomServer) handleUserInfo(w http.ResponseWriter, r *http.Request) {
 	}
 	userID := ti.GetUserID()
 
-	p := bluemonday.UGCPolicy()
-	ava := p.Sanitize(fmt.Sprintf(c.URL+"/avatar?user=%s", userID))
-	res := fmt.Sprintf(`{
-					"id": "%s",
-					"name":"%s",
-					"picture":"%s"
-					}`, userID, userID, ava)
+	user := token.User{
+		ID:      userID,
+		Name:    userID,
+		Picture: fmt.Sprintf(c.URL+"/avatar?user=%s", userID),
```

Comment:
vulnerability fixed: backslashes and double quotes in `userID` were not being escaped here (double quotes were _explicitly_ not escaped)